cmd/openshift-install/main.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"golang.org/x/crypto/ssh/terminal"
-	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
```

Comment:
so the [vendor update](https://www.youtube.com/watch?v=KR0g-1hnQPA) doesn't reflect that klog was dropped or not used anymore by anyone.

so how can ensure no  logs end up into stderr from both versions

Reply:
I can use here both klog versions, do you think this is better approach?

Reply:
> I can use here both klog versions, do you think this is better approach?

i think that would be safer for now.

Reply:
Thanks @abhinavdahiya 
I changed it, please review